### ISSUE TITLE
Add a close button to an image.

### ISSUE THREAD

**Issue by ctarx** [spikecodes/libreddit#382]
## How does this feature work on Reddit?

  Reddit has a close button when you click on a picture, so you can easly come back from a full size picture into a subreddit.

## Describe the implementation into Libreddit

 It can be just a simple close button made in html/css

## Describe alternatives you've considered

 Or maybe additional click on a picture can close it and move as back to the sub

## Additional context

  
![Screenshot from 2022-01-02 21-09-12](https://user-images.githubusercontent.com/5619861/147889555-9e08a8d7-a3c6-45a9-912f-d6c5cd2372d3.png)

**Comment by Nachtalb** [comment]
The images are opened directly via the URL and handled by the browser, not in a popup. There is no way to add a close button there. Just go back with <kbd>ALT</kbd>+<kbd>←</kbd>. You can also open the images by clicking them with the middle mouse key to open them in a new tab and then close the tab with <kbd>CTRL</kbd>+<kbd>w</kbd>, at least that's how I do it.

**Comment by ctarx** [comment]
Yes I do and it's not a problem but on iOS, where I made a home screen shortcut,  I don't have any icons/arrows on the bottom so whenever I tap on the sub or picture I have to close an app and open again.

**Comment by mikupls** [comment]
I think you're both right. I'm not sure how to solve this since we are opening the image "directly". I also tried to use libreddit on iOS recently, and ran into the same problem, making libreddit mostly unusable.
